packages/apps/dev-wallet/src/Components/PersonalizeProfile/PersonalizeProfile.tsx

```diff
@@ -0,0 +1,82 @@
+import {
```

Comment:
lets put this file next to the page in the create-profile folder

Reply:
Done